docs/user-guide/systemrequirements.md

```diff
@@ -59,8 +59,10 @@ For more information on the relationship between the Zowe server components and
 
 ### Zowe Web Explorers and APIs on z/OS requirements (host)
 
-- Node.js v8.x (except v8.16.1), and v12.x.
-  **Note:** when using v12.x, it is highly recommended that plug-ins used are tagged. For more information, see [Tagging on z/OS](../extend/extend-desktop/mvd-buildingplugins.md#tagging-plugin-files-on-z-os)
+- Node.js v8.x (except v8.16.1), v12.x, or v14.
+  
+  **Note:** When using Node.js v12.x, it is highly recommended that plug-ins used are tagged. For more information, see [Tagging on z/OS](../extend/extend-desktop/mvd-buildingplugins.md#tagging-plugin-files-on-z-os).
```

Comment:
tagging will apply to every version v12 and afterward, such as v14 and whatever version comes after v14.

Reply:
Fixed. Now it says `When using Node.js v12.x or later`